s4e-web/src/app/views/map-view/sentinel-search/search-result-modal/search-result-modal.component.html

```diff
@@ -2,7 +2,7 @@
 <button [disabled]="isLast$ | async" (click)="nextResult()" class="modal__next"><i class="fa fa-chevron-right"></i></button>
 <s4e-generic-modal [buttonX]="true" [modalId]="registeredId">
   <div class="s4e-modal-header">
-    {{(searchResult$ | async)?.id}}
+    {{(searchResult$ | async)?.sceneKey}}
```

Comment:
I think the value displayed here should be only the last part of the path, excluding `.scene`, as discussed.
For example: /MSG_Products_WM/OST/20200201/**202002010000_WV-IR**.scene (the bold part should be retained)